docs/source/Northeastern-Cluster.md

```diff
@@ -1,48 +1,48 @@
 # Northeastern Cluster
-Instructions for [Accessing Northeastern Cluster](Accessing-Northeastern-Cluster.html)
+Instructions for [Accessing Northeastern Cluster](clusters/kaizen/Accessing-Northeastern-Cluster.html)
 
 ### Projects
-* [Kaizen Production](Kaizen-Production.html)
-* [Monitoring in NEU Env.](Monitoring-in-NEU-Env..html)
-* [Staging Area](Staging-Area.html)
-* [VMs running on physical nodes](VMs-running-on-nodes.html)
+* [Kaizen Production](clusters/kaizen/Kaizen-Production.html)
+* [Monitoring in NEU Env.](clusters/kaizen/Monitoring-in-NEU-Env..html)
```

Comment:
Can we rename `Monitoring in NEU Env.` to `Monitoring in NEU Env`? That `..html` originally made me think the link was bad.

Reply:
Sure